client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/Formatters.spec.ts

```diff
@@ -20,46 +20,20 @@ describe("Text Message Formatter", () => {
     });
 
     ([
-        ["TABC", new Error("Invalid length: 'ABC'")],
-        ["X1:T:A", new Error("Unsupported message format: 'X'")],
-        ["T1:T:A;12ab34:", new Error("Invalid length: '12ab34'")],
-        ["T1:T:A;1:asdf:", new Error("Unknown type value: 'asdf'")],
-        ["T1:T:A;1::", new Error("Message is incomplete")],
-        ["T1:T:A;1:AB:", new Error("Message is incomplete")],
-        ["T1:T:A;5:T:A", new Error("Message is incomplete")],
-        ["T1:T:A;5:T:AB", new Error("Message is incomplete")],
-        ["T1:T:A;5:T:ABCDE", new Error("Message is incomplete")],
-        ["T1:T:A;5:X:ABCDE", new Error("Message is incomplete")],
-        ["T1:T:A;5:T:ABCDEF", new Error("Message missing trailer character")],
+        ["ABC", new Error("Invalid length: 'ABC'")],
+        ["1:X:A", new Error("Message is incomplete")],
```

Comment:
Why is this 'Message is incomplete'? Isn't X supposed to be message type in which case it would be invalid?